go.mod

```diff
@@ -16,6 +17,7 @@ require (
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
 	github.com/uber/tchannel-go v1.16.0
 	go.uber.org/atomic v1.7.0
+	go.uber.org/fx v1.10.0 // indirect
```

Comment:
Just curious, why the fx gets introduced here?

Reply:
This is the first time use of generated proto types. That package also includes YARPC interfaces which in turn references fx.